doc/release-notes.md

```diff
@@ -4,3 +4,51 @@ release-notes at release time)
 Notable changes
 ===============
 
+Sprout to Sapling Migration Tool
+--------------------------------
+This release includes the addition of a tool that will enable users to migrate
+shielded funds from the Sprout pool to the Sapling pool while minimizing
+information leakage. 
+
+The migration can be enabled using the RPC `z_setmigration` or by including
+`migration=1` in the `zcash.conf` file. Unless otherwise specified funds will be
+migrated to the wallet's default Sapling address; it is also possible to set the 
+receiving Sapling address using the `migrationdestaddress=<zaddr>` option in
+`zcash.conf`.
+
+See [ZIP308](https://github.com/zcash/zips/blob/master/zip-0308.rst) for full details.
+
+Sprout to Sapling Migration Tool Fixes
+--------------------------------------
+The 2.0.5-1 and 2.0.5-2 releases include fixes to the Sprout to Sapling
+Migration Tool found in testing. For a complete list see the [2.0.5 milestone](https://github.com/zcash/zcash/milestone/79?closed=1)
```

Comment:
The link is for closed tickets in the 2.0.5 milestone, which is useful but does not correspond to the text.

Reply:
```suggestion
Migration Tool found in testing.

For a complete list of changes in 2.0.5, 2.0.5-1 and 2.0.5-2, see the [2.0.5 milestone](https://github.com/zcash/zcash/milestone/79?closed=1).
```